_posts/2020/2020-01-19-recruitment-final-list-20.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title: Final recruitment list For 1st year (2019)
+excerpt: Recruitment results 2019.
+---
+
+We interviewed 80 people before releasing the final list of selected people.
+It was a really tough job to filter down from the talented applicants to the
+final list. Sadly, we have limited vacancies and can only take in a limited
```

Comment:
Unnecessary linebreaks are there. Keep the entire paragraph in 1 line so that the text doesn't appear broken